src/views/PropertyView.vue

```diff
@@ -0,0 +1,235 @@
+          router.push(`/project/${props.projectId}/objects`);
+        })
+        .catch((err) => {
+          console.error("Error deleting property:", err);
+        });
+    };
+
+    const close = () => {
+      router.push(`/project/${props.projectId}/objects`);
+    };
+
+    return {
+      id,
+      title,
+      description,
+      landRegisterEntry,
+      effectiveSpace,
+      plotArea,
+      createProperty,
+      updateProperty,
+      deleteProperty,
+      fetchPropertyDetails,
+      close,
+      error,
+      isLoading,
+      route,
+      action,
+    };
+  },
+};
+</script>
+
+<template>
+  <div class="col-12">
+    <div class="card">
+      <h5 v-if="action === 'create'">Erstelle Eigentum</h5>
+      <h5 v-if="action === 'update'">Update Eigentum mit ID: {{ id }}</h5>
+      <h5 v-if="action === 'delete'">Lösche Eigentum mit ID: {{ id }}</h5>
+      <div v-if="isLoading">Loading...</div>
+      <div v-if="error">{{ error }}</div>
+      <div v-if="!isLoading && !error" class="p-fluid formgrid grid">
+        <div class="field col-12 md:col-6">
+          <label for="title">Title</label>
+          <InputText
+            v-model="title"
+            id="title"
+            type="text"
+            :disabled="action === 'delete'"
+          />
+        </div>
+        <div class="field col-12 md:col-6">
+          <label for="landRegisterEntry">Grundbucheintrag</label>
+          <InputText
+            v-model="landRegisterEntry"
+            id="landRegisterEntry"
+            type="text"
+            :disabled="action === 'delete'"
+          />
+        </div>
+        <div class="field col-12">
+          <label for="description">Beschreibung</label>
+          <Textarea
+            v-model="description"
+            id="description"
+            rows="4"
+            :disabled="action === 'delete'"
+            class="no-resize"
+          />
+        </div>
+        <div class="field col-10 md:col-6">
+          <label for="plotArea">Grundstücksfläche</label>
+          <InputText
+            v-model="plotArea"
+            id="plotArea"
+            type="number"
+            :disabled="action === 'delete'"
+          />
+        </div>
+        <div class="field col-10 md:col-6">
+          <label for="effectiveSpace">Nutzfläche</label>
+          <InputText
+            v-model="effectiveSpace"
+            id="effectiveSpace"
+            type="number"
+            :disabled="action === 'delete'"
+          />
+        </div>
+        <div class="field col-12 text-right">
+          <Button
+            v-if="action === 'create'"
+            label="Erstellen"
+            icon="pi pi-check"
+            @click="createProperty"
+            class="p-button-property mr-2 mb-2"
+          />
+          <Button
+            v-if="action === 'update'"
+            label="Updaten"
+            icon="pi pi-check"
+            @click="updateProperty"
+            class="p-button-property mr-2 mb-2"
+          />
```

Comment:
for qol you could notify the user that updating the data worked correctly
this also applies for all the other buttons

Reply:
Great Idea. This should be implemented